src/main/java/spring/boot/parser/model/User.java

```diff
@@ -0,0 +1,43 @@
+package spring.boot.parser.model;
+
+import java.util.Set;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+
+@AllArgsConstructor
+@Builder
+@Data
+@EqualsAndHashCode(of = {"userId"})
+@NoArgsConstructor
+@Entity
+@Table(name = "users")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @NonNull
+    @Column(name = "user_id", unique = true)
+    private String userId;
+
+    @Column(name = "profile_name")
+    private String profileName;
+
+    @ManyToMany
+    @Column(name = "roles")
+    private Set<Role> role;
+
+    @Column(name = "passwords")
+    private String password;
```

Comment:
```suggestion
    @ManyToMany
    private Set<Role> role;
    
    private String password;
```
is there a separate table created to store user-role relationship? if yes, do we need @Column annotation?

Reply:
I do put annotation for all fields to keep the unity of the code. I mean all fields must be very similar, not like here we put annotation there may be too. 
Yes, there are different tables for the user- role relationship.
But there no users :) bug! )